MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs

```diff
@@ -645,6 +660,35 @@ private static DataFeedEnum<T> GenerateEnumField<T>(IReadOnlyList<string> path,
             return enumField;
         }
 
+        private static IEnumerable<DataFeedItem> GenerateFlagsEnumFields<T>(IReadOnlyList<string> path, IDefiningConfigKey<T> configKey)
+            where T : Enum
+        {
+            var flagsEnumGroup = new DataFeedGroup();
+            flagsEnumGroup.InitBase(configKey.FullId, path, [configKey.Section.Id], configKey.GetLocaleKey("Name").AsLocaleKey());
+            flagsEnumGroup.InitDescription(configKey.GetLocaleKey("Description").AsLocaleKey());
+            yield return flagsEnumGroup;
+
+            var flagsGrouping = new[] { configKey.Section.Id, configKey.FullId };
+
+            foreach (var value in Enum.GetValues(configKey.ValueType).Cast<T>())
+            {
+                var name = value.ToString();
+                var longValue = Convert.ToInt64(value);
+
+                var flagToggle = new DataFeedToggle();
+                flagToggle.InitBase($"{configKey.FullId}.{name}", path, flagsGrouping, name);
+                flagToggle.InitSetupValue(field =>
```

Comment:
This could be way nicer, it was just the basic test now - and atm disabling a member that's part of an active combination member also disables all other members that are part of it, due to change chaining